docs/content/latest/develop/query-1-performance/explain-analyze.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+linkTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+description: Query optimization with EXPLAIN and EXPLAIN ANALYZE
+aliases:
+  - /develop/query-1-performance/explain-analyze/
+headerTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+image: /images/section_icons/index/develop.png
+menu:
+  latest:
+    identifier: explain-analyze
+    parent: query-1-performance
+    weight: 566
+isTocNested: true
+showAsideToc: true
+---
+
+This section describes how to optimize queries using YSQL's `EXPLAIN` and `EXPLAIN ANALYZE` statements.
+
+## The EXPLAIN Statement
+
+Using the `EXPLAIN` statement, you can obtain the execution plan generated by YSQL for each received query. This statement returns the following two pieces of information: 
```

Comment:
generated by YSQL for each received query.
-->
for a given SQL statement.

This statement returns
-->
This statement also returns

the following two pieces of information
-->
the following information

Reply:
Implemented review comments.
Merging the PR.